static/src/js/containers/AdminPreferencesMetricsContainer/AdminPreferencesMetricsContainer.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import { withRouter } from 'react-router-dom'
+
+import actions from '../../actions'
+import AdminPreferencesMetrics from '../../components/AdminPreferencesMetrics/AdminPreferencesMetrics'
+
+export const mapStateToProps = (state) => ({
+  metricsPreferences: state.admin.metricsPreferences,
+  preferencesLoading: state.admin.metricsPreferences.isLoading,
```

Comment:
The loading/loaded props aren't being used. You could either remove them from `mapStateToProps` and in the redux store, or add loading indicators to the UI.